contrib/docker_compose/docker_cmd.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# echo "ccs:x:$(id -u):$(id -g):Calendar and Contacts Server:/opt/ccs:/bin/bash" >> /etc/passwd
+
+# Just get our conf file
+CCS_CONF_TEMP_FILE="/opt/ccs/caldavd.plist.template"
+
+# It is important that this dir is world-writable,
+# /tmp usually is
+CCS_CONF_FILE="/etc/caldavd/caldavd.plist"
+CCS_AUTH_FILE="/etc/caldavd/accounts.xml"
+
+# create SQL tables on first run
+echo $(awk -F: '{print $2":"$3}' <<< $POSTGRES_HOST):$POSTGRES_DB:$POSTGRES_USER:$POSTGRES_PASS > ~/.pgpass
+chmod 600 ~/.pgpass
+
+# wait for postgres to be available
+export PGHOST=$(awk -F: '{print $2}' <<< $POSTGRES_HOST)
+until psql -h $PGHOST -U $POSTGRES_USER $POSTGRES_DB -c '\q'; do
+  >&2 echo "Postgres is unavailable - sleeping"
+  sleep 1
+done
+
+TD=$(psql -h $PGHOST -U $POSTGRES_USER $POSTGRES_DB <<< '\dt')
+
+if [ "$TD" = "No relations found." ]; then
+    psql -h $PGHOST -U $POSTGRES_USER $POSTGRES_DB < /opt/ccs/txdav/common/datastore/sql_schema/current.sql
+fi
+
+# create config file on first run
+if [ ! -f $CCS_CONF_FILE ]; then
+    envsubst < $CCS_CONF_TEMP_FILE > $CCS_CONF_FILE
+fi
+
+# create auth file if not available
+if [ ! -f $CCS_AUTH_FILE ]; then
+    cp /opt/ccs/conf/auth/accounts.xml $CCS_AUTH_FILE
+    cp /opt/ccs/conf/auth/*.dtd /etc/caldavd
+fi
+
+# Run caldavd, no daemonize, log to stdout
+caldavd -X -L -f $CCS_CONF_FILE
```

Comment:
You mean having `caldavd` as the default command? What would be the benefit?

Reply:
Didn't think about this entirely yet, but caldavd also provides some management utilities in the same package. Basically by having that preconfiguration done in the entrypoint means one may run a command of choice instead of just the daemon.

It's implemented that way here:
https://github.com/giorgioazzinnaro/ccs-calendarserver/blob/master/contrib/docker/docker_entrypoint.sh